front/src/modules/trainschedule/components/ManageTrainSchedule/PacedTrainSettings/AddedOccurences.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+
+import { Clock, Calendar, Trash, Dot } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { addException, deleteException } from 'reducers/osrdconf/operationalStudiesConf';
+import { getExceptions } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { useAppDispatch } from 'store';
+
+const AddedOccurences = () => {
+  const [date, setDate] = useState('');
+  const [time, setTime] = useState('');
+  const exceptions = useSelector(getExceptions);
+  const { t } = useTranslation('operational-studies', { keyPrefix: 'manageTrainSchedule' });
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    const now = new Date();
+    setDate(now.toISOString().slice(0, 10));
```

Comment:
`toISOString()` will format the date in UTC. That is, in France, if the current day is close to midnight, the function will give the previous or next day instead of the current one. (In other countries the function may misbehave more often.)

Reply:
Note, we already have `formatLocalDate()` for the purpose of feeding data into `<input type="date">`.

Reply:
right, I missed that !

Reply:
done